sdcm/cluster.py

```diff
@@ -3064,6 +3064,8 @@ def get_group0_members(self) -> list[dict[str, str]]:
             self.log.error(err_msg)
             InfoEvent(message=err_msg, severity=Severity.ERROR).publish()
 
+            raise exc
```

Comment:
AFAIK you can just:
```suggestion
            raise
```

Reply:
> little comment, and there are checks that failed, but other than that, LGTM

i re-triggered the checks, as the precommit one seems to be failed on an environment error, and may go away when re-running it

Reply:
> AFAIK you can just:

np, changed this